contracts/eip20/EIP20.sol

```diff
@@ -1,14 +1,18 @@
 /*
 Implements EIP20 token standard: https://github.com/ethereum/EIPs/issues/20
 .*/
-pragma solidity ^0.4.8;
+
+
+pragma solidity 0.4.18;
```

Comment:
I know pinning is the best practice, but we do run into a practical concern here. When consuming this from EPM, the user is unlikely to have the exact same solc we tested with (because of, uh, the nature of time, theirs is probably newer). Furthermore, if they are consuming other EPM packages that pin a different, incompatible pragma, they simply can't compile. I feel that in EPM, unpinned pragmas are a necessary evil. Do let me know your thoughts.

Reply:
This came up recently here: https://github.com/ConsenSys/smart-contract-best-practices/issues/125

So I see your point, and think it makes sense to let it float.